### ISSUE TITLE
Lua error messages can to lead asserts in fmt library

### ISSUE THREAD

**Issue by AchimStremplat** [bmwcarit/ramses-logic#41]
An attempt to call LogicEngine::createLuaModule with the following (erroneous) source code causes in debug an assert in the format library:

```
local coalaModule = {}

coalaModule.coalaStruct = {
	oink1
	oink2
}

return coalaModule
```

The problem seems to be that the error message returned by Lua contains curly brackets, which are then interpreted by the format library. Maybe there are also other character sequences which could cause the fmt library to do odd stuff?

**Comment by violinyanev** [comment]
Hi @AchimStremplat ! That is definitely a bug, will be fixed in the next patch release. Thanks for reporting!

**Comment by violinyanev** [comment]
Fixed in v1.0.1
